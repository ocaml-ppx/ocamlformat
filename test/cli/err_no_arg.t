  $ ocamlformat
  ocamlformat: Must specify at least one input file, or `-` for stdin
  [1]
