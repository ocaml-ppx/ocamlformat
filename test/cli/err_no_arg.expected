ocamlformat: Must specify at least one input file, or `-` for stdin
